backend/src/share/share.controller.ts

```diff
@@ -78,6 +78,20 @@ export class ShareController {
     );
   }
 
+  @Delete(":id/complete")
+  @UseGuards(ShareOwnerGuard)
+  async revertComplete(@Param("id") id: string) {
+    return new ShareDTO().from(
+      await this.shareService.revertComplete(id),
+    );
+  }
+
+  @Delete(":id")
+  @UseGuards(JwtGuard, ShareOwnerGuard)
```

Comment:
We can remove `JwtGuard` here because `ShareOwnerGuard` extends it now
```suggestion
  @UseGuards(ShareOwnerGuard)
```